console-extensions.json

```diff
@@ -78,5 +78,12 @@
       "section": "observe",
       "insertAfter": "dashboards"
     }
+  },
+	{
```

Comment:
Nit: Looks like a tab character instead of 2 space characters

Reply:
Removed mocked extension in latest commit.